test/tags/mix.exs

```diff
@@ -0,0 +1,90 @@
+# https://github.com/elixir-mint/mint/blob/e7199d1163e5a8a1b0001cacc93063e866dbaa35/mix.exs
+defmodule Mint.MixProject do
```

Comment:
I think it would make more sense to come up with minimal examples matching the queries in tags.scm. Something like [here](https://github.com/elixir-lang/tree-sitter-elixir/blob/a11a686303355a518b0a45dea7c77c5eebb5ec22/test/highlight/module.ex), this way we avoid duplication and if something fails it should be more clear. We can keep this file under `tags/integration` if you like :)

From edge cases, we are missing `defimpl`, function call as in `|> foo` and maybe another definition keyword like `defmacro`.

Reply:
Welp I found a bug in my PR :sweat_smile:

Looks like you can't have directories under `test/{highlight,tags}` - it fails trying to read the extension to determine which parser to use. I'll set myself a reminder to submit a patch for that! In the meantime I've removed `test/tags/mix.exs`. I think the new module.ex tests cover everything we saw in `mix.exs` in a more compact way.